functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SkipTestExecutionFunctionalTest.java

```diff
@@ -72,4 +78,97 @@ public void should_disable_smart_testing_and_execute_no_tests_when_test_executio
         assertThat(capturedMavenLog).contains(SMART_TESTING_EXTENSION_DISABLED);
         assertThat(actualTestResults.accumulatedPerTestClass()).size().isEqualTo(0);
     }
+
+    @Test
+    public void should_disable_smart_testing_and_execute_no_tests_when_skip_test_is_set_as_default_in_pom() throws Exception {
+        // given
+        final Project project = testBed.getProject();
+
+        project
+            .applyAsCommits("Configures skipTests as default property in pom");
+
+        project.configureSmartTesting()
+                .executionOrder(AFFECTED)
+                .inMode(ORDERING)
+            .enable();
+
+        // when
+        final TestResults actualTestResults = project.build(CORE_MODULES).run();
+
+        // then
+        String capturedMavenLog = project.getMavenLog();
+        assertThat(capturedMavenLog).contains(SMART_TESTING_EXTENSION_DISABLED);
+        assertThat(actualTestResults.accumulatedPerTestClass()).size().isEqualTo(0);
+    }
+
+    @Test
+    public void should_disable_smart_testing_and_execute_no_tests_when_skip_is_set_in_plugin_configuration_section_of_pom() throws Exception {
+        // given
+        final Project project = testBed.getProject();
+
+        project
+            .applyAsCommits("Configure skipTests property in plugin configuration section of pom");
+
+        project.configureSmartTesting()
+                .executionOrder(AFFECTED)
+                .inMode(ORDERING)
+            .enable();
+
+        // when
+        final TestResults actualTestResults = project.build(CORE_MODULES).run();
+
+        // then
+        String capturedMavenLog = project.getMavenLog();
+        assertThat(capturedMavenLog).contains(SMART_TESTING_EXTENSION_DISABLED);
+        assertThat(actualTestResults.accumulatedPerTestClass()).size().isEqualTo(0);
+    }
+
+    @Test
+    public void should_execute_only_unit_tests_when_skipITs_is_set_in_pom() throws Exception {
+        // given
+        final Project project = testBed.getProject();
+
+        project
+            .applyAsCommits("Configure skipITs as default property in pom");
+
+        project.configureSmartTesting()
+                .executionOrder(AFFECTED)
+                .inMode(ORDERING)
+            .enable();
+
+        // when
+        final TestResults actualTestResults = project.build(CORE_MODULES).run("clean", "verify");
+
+        // then
+        String capturedMavenLog = project.getMavenLog();
+        assertThat(capturedMavenLog).contains(SMART_TESTING_EXTENSION_ENABLED);
+        assertThat(actualTestResults.accumulatedPerTestClass()).size().isEqualTo(20);
+    }
+
+    @Test
+    public void should_be_able_to_enable_tests_from_property_when_set_as_default_property_in_pom() throws Exception {
+        // given
+        final Project project = testBed.getProject();
+
+        project
+            .applyAsCommits("Configure skipITs as default property in pom");
```

Comment:
same as for the surefire one:
```xml
+    <skipITs>true</skipITs>
   </properties>
 
   <issueManagement>
@@ -87,10 +90,31 @@
             <include>**/*TestCase.java</include>
             <include>**/*TestSuite.java</include>
           </includes>
+          <excludes>
+            <exclude>**/IT*.java</exclude>
+          </excludes>
           <useFile>true</useFile>
         </configuration>
       </plugin>
       <plugin>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <configuration>
+          <skipITs>${skipITs}</skipITs>
```
You just evaluate the property you have set in the properties part... The property can be also 
`<my.cool.skip>true</my.cool.skip>`
and evaluating it: 
`<skipITs>${my.cool.skip}</skipITs>`
would have the very same effect

Reply:
Updated the surefire tag and leaving this one as it is, because it also verifies another case, https://github.com/arquillian/smart-testing/pull/211/files#diff-d7b0ca86a6c38fc16017af7fe32281a1R149, i.e user can reset the property from system property when set in property section of the pom.xml.

Reply:
Then change the property name to contain something else, otherwise, you cannot verify it correctly.

Reply:
I get your point to rename property name to something like `<my.cool.skip>true</my.cool.skip>` but I wanted to stick with the official documentation, so I was trying to test the case `Skipping By Default` as per [this](http://maven.apache.org/surefire/maven-failsafe-plugin/examples/skipping-tests.html).
Having the property set as `<my.cool.skip>true</my.cool.skip>` in my opinion was almost similar to having it set inside the plugin configuration, so I excluded this case.

Reply:
Updated.